docs/src/extend/custom-processors.md

```diff
@@ -81,7 +82,9 @@ By default, ESLint will not perform autofixes when a processor is used, even whe
 
 2. Add a `supportsAutofix: true` property to the processor.
 
-You can have both rules and processors in a single plugin. You can also have multiple processors in one plugin.
+TODO: i don't understand what this next paragraph means
+
+You can have both rules and custom processors in a single plugin. You can also have multiple processors in one plugin.
```

Comment:
To reviewer: I have trouble understanding this line. Could you explain more?

Reply:
We are trying to say that a single plugin can export rules, processors, or both. In other words, you don't need one plugin to distribute rules and another to distribute processors, you can just put them in the same one.

Reply:
> To support multiple extensions, add each one to the `processors` element and point them to the same object.

specifically this part of that paragraph i am still having trouble understanding. could you explain a bit more?

Reply:
Suppose you want the same processor to work on `.md` and `.mdx` files, you might end up configuring your plugin like this:

```js
const yourProcessor = {};

module.exports = {
    processors: {
        ".md": yourProcessor,
        ".mdx": yourProcessor
    }
};
```

Reply:
ah got it! adding new paragraph and small example